src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentInformationTable.tag

```diff
@@ -0,0 +1,215 @@
+            <div class="col-sm-11"
+                id="<%= Const.ParamsNames.STUDENT_EMAIL %>">
+                <p class="form-control-static">${studentInfoTable.email}</p>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-sm-1 control-label">Comments:</label>
+            <div class="col-sm-11"
+                id="<%= Const.ParamsNames.COMMENTS %>">
+                <p class="form-control-static">${studentInfoTable.comments}</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="commentArea" class="well well-plain" style="display: none;">
+    <form method="post" action="<%= Const.ActionURIs.INSTRUCTOR_STUDENT_COMMENT_ADD %>" name="form_commentadd">
+        <div class="form-group form-inline">
+            <label style="margin-right: 24px;">Recipient:
+            </label> 
+            <select id="comment_recipient_select" class="form-control">
+                <option value="<%= CommentParticipantType.PERSON %>" selected>${studentInfoTable.name}</option>
+                <option value="<%= CommentParticipantType.TEAM %>">${studentInfoTable.team}</option>
+                <c:if test="${studentInfoTable.hasSection && studentInfoTable.section != 'None'}">
+                    <option value="<%= CommentParticipantType.SECTION %>">${studentInfoTable.section}</option>
+                </c:if>
+            </select>
+            <a id="visibility-options-trigger"
+                class="btn btn-sm btn-info pull-right"><span
+                class="glyphicon glyphicon-eye-close"></span>
+                Show Visibility Options</a>
+        </div>
+        <p class="form-group text-muted">
+            The default visibility for your comment is private. You may change it using the visibility options above.
+        </p>
+        <div id="visibility-options" class="panel panel-default"
+            style="display: none;">
+            <div class="panel-heading">Visibility Options</div>
+            <table class="table text-center"
+                style="background: #fff;">
+                <tbody>
+                    <tr>
+                        <th class="text-center">User/Group</th>
+                        <th class="text-center">Can see your comment</th>
+                        <th class="text-center">Can see giver's name</th>
+                        <th class="text-center">Can see recipient's name</th>
+                    </tr>
+                    <tr id="recipient-person">
+                        <td class="text-left">
+                            <div data-toggle="tooltip"
+                                data-placement="top" title=""
+                                data-original-title="Control what comment recipient(s) can view">
+                                Recipient(s)</div>
+                        </td>
+                        <td><input
+                            class="visibilityCheckbox answerCheckbox centered"
```

Comment:
`input` can be on a new line and the first attribute (in this case, `class`) can be on the same line as `input` itself.